cmd/create/machinepool/cmd.go

```diff
@@ -198,9 +222,49 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Autoscaling
+	autoscaling := args.autoscalingEnabled
+	if !autoscaling && interactive.Enabled() {
```

Comment:
Most of my thoughts on this file are similar to the `create cluster` one

Reply:
I added validations on both